src/assets/styles/header.scss

```diff
@@ -0,0 +1,279 @@
+.header {
+  background: $dark;
+  text-align: center;
+  padding: 1em;
+}
+
+.header-inner {
+  margin: 0 auto;
+  width: 80%;
+  max-width: 500px;
+}
+
+.toolbar {
+  position: relative;
+}
+
+.toolbar.-desktop {
+  display: none;
+}
+
+.logo {
+  max-width: 210px;
+}
+
+.cta {
+  font-family: "Freight Sans CMP";
```

Comment:
I would put this in variables.scss since we are repeating it so much. something like `$accent-font: "Freight Sans CMP";`